packages/web-components/src/card/card.styles.ts

```diff
@@ -0,0 +1,167 @@
+import { css } from '@microsoft/fast-element';
+import { display } from '@microsoft/fast-foundation';
+import {
+  borderRadiusMedium,
+  colorNeutralBackground1,
+  colorNeutralBackground1Hover,
+  colorNeutralBackground1Pressed,
+  colorNeutralBackground1Selected,
+  colorNeutralBackground2,
+  colorNeutralBackground2Hover,
+  colorNeutralBackground2Pressed,
+  colorNeutralBackground2Selected,
+  colorNeutralBackgroundDisabled,
+  colorNeutralStroke1,
+  colorNeutralStroke1Hover,
+  colorNeutralStroke1Pressed,
+  colorNeutralStroke1Selected,
+  colorSubtleBackground,
+  colorSubtleBackgroundHover,
+  colorSubtleBackgroundPressed,
+  colorSubtleBackgroundSelected,
+  colorTransparentBackground,
+  colorTransparentBackgroundHover,
+  colorTransparentBackgroundPressed,
+  colorTransparentBackgroundSelected,
+  colorTransparentStroke,
+  colorTransparentStrokeDisabled,
+  colorTransparentStrokeInteractive,
+  fontFamilyBase,
+  shadow4,
+  shadow8,
+  spacingHorizontalL,
+  spacingHorizontalM,
+  spacingHorizontalS,
+  spacingVerticalL,
+  spacingVerticalM,
+  spacingVerticalS,
+  strokeWidthThin,
+} from '../theme/design-tokens.js';
+
+/** Card styles
+ * @public
+ */
+export const styles = css`
+  ${display('grid')}
+
+  :host {
+    font-family: ${fontFamilyBase};
+    background: ${colorNeutralBackground1};
+    border: ${strokeWidthThin} solid ${colorTransparentStroke};
+    border-radius: ${borderRadiusMedium};
+    position: relative;
+    max-width: 100%;
+    height: fit-content;
+    margin: auto;
+    overflow: hidden;
+    box-shadow: ${shadow4};
+    row-gap: ${spacingVerticalM};
+    column-gap: ${spacingHorizontalM};
+    padding: ${spacingVerticalM} ${spacingHorizontalM};
+    row-gap: 12px;
+    column-gap: 12px;
+    padding: 12px;
```

Comment:
These are duplicated from above. Should we use `12px` or the design tokens?

Reply:
Yes, these are going to take precedence over the above - should these be the tokens?